payjoin/src/persist.rs

```diff
@@ -1115,6 +1109,109 @@ mod tests {
         assert_eq!(no_results.success(), None);
     }
 
+    #[test]
```

Comment:
Are these tests specific for the receiver state machine? If so lets move them to receive::v2::mod::test.